dali/test/python/test_dali_tf_dataset_mnist.py

```diff
@@ -290,14 +292,16 @@ def _test_estimators_wrapping_keras_single_device(device='cpu', device_id=0):
         device_id)
 
 
-@with_setup(clear_checkpoints, clear_checkpoints)
-def test_estimators_wrapping_keras_single_gpu():
-    _test_estimators_wrapping_keras_single_device('gpu', 0)
+# Temporarily disabled, waiting for TF updates
+# @with_setup(clear_checkpoints, clear_checkpoints)
+# def test_estimators_wrapping_keras_single_gpu():
+#     _test_estimators_wrapping_keras_single_device('gpu', 0)
 
 
-@with_setup(clear_checkpoints, clear_checkpoints)
-def test_estimators_wrapping_keras_single_other_gpu():
-    _test_estimators_wrapping_keras_single_device('gpu', 1)
+# Temporarily disabled, waiting for TF updates
+# @with_setup(clear_checkpoints, clear_checkpoints)
+# def test_estimators_wrapping_keras_single_other_gpu():
+#     _test_estimators_wrapping_keras_single_device('gpu', 1)
```

Comment:
Removed tests would worked without the assertion added in this PR, but they would copy DALI GPU output to TF CPU buffer. 9.9/10 this is not what we want at all.